packages/jsii-dotnet-runtime-test/test/Amazon.JSII.Runtime.IntegrationTests/Amazon.JSII.Runtime.IntegrationTests.csproj

```diff
@@ -19,4 +19,8 @@
         <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
     </ItemGroup>
 
+    <ItemGroup>
+      <ProjectReference Include="..\..\..\jsii-dotnet-runtime\src\Amazon.JSII.Runtime\Amazon.JSII.Runtime.csproj" />
+    </ItemGroup>
```

Comment:
where is this coming from and how come it wasn't required so far?

Reply:
So. I've added this because it makes working with the integration tests in an IDE much nicer... It isn't _required_ per se, but ensures that the *local* source is used instead of what's in the symbols package (makes debugging more efficient, etc...)

I routinely add this when I work in the kernel, and since this is not a published artifact, I figured I'd leave it in this time... Because it'd help everyone in this situation.